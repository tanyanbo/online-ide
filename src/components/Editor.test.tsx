import { render, screen } from "react-dom";
import React from "react";

test("testing", () => {});
