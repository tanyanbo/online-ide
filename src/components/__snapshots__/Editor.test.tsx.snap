// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Editor/> users can type into the codemirror window 1`] = `
Array [
  <div
    className="editor"
  >
    <div
      className="editor-top"
    >
      <h3>
        HTML
      </h3>
      <button
        className="reset-button"
        onClick={[Function]}
      >
        ❌
      </button>
    </div>
    <div
      className="react-codemirror2 code-mirror"
    />
  </div>,
  <div
    className="buttons"
  >
    <form
      className="run-checkbox"
    >
      <input
        id="run"
        onChange={[Function]}
        type="checkbox"
      />
      <label
        htmlFor="run"
      >
        边写边运行
      </label>
    </form>
    <button
      className="run-button"
      disabled={false}
      onClick={[Function]}
    >
      运行
    </button>
  </div>,
  <iframe
    height="500px"
    sandbox="allow-scripts allow-same-origin allow-modals"
    srcDoc="
          <html lang=\\"en-US\\">
          <head>
            <title>Iframe</title>
            <style>
              undefined
            </style>
          </head>
          <body>
            
            <script>
              try {
                
              } catch(e) {
                throw new Error('custom error');
              }
            </script>
          </body>
        </html>
      "
    title="output"
    width="100%"
  />,
]
`;
